app/views/events/participants.html.haml

```diff
@@ -0,0 +1,24 @@
+.col-md-12
+  %h3 Participants of #{@event.name}
+  %br
+  %table.table
+    %tr
+      %th #
+      %th Nickname
+      %th Email
+      %th State
+      %th Role
+      %th Change role to event organizer
+    - @event.users.includes(:profile).distinct.each.with_index(1) do |user,index|
```

Comment:
But [user requests](https://github.com/openSUSE/travel-support-program/pull/80/files/98500327a95b33f18b94f0ed1915d081e1b02add#diff-018ebd24bb84fbd780e127372855d5c5R20) would result in another db query, right? Including the requests before would cause rails to preload them and thus reduce the number of queries.